### PR TITLE
Grouped Parabola and PingPong. Also added a new Ease -- the Fancy™SineWave!

### PR DESCRIPTION
**Pull request by FancySensei**
The popularity of the Parabola (`0->1->0`) easing functions are growing -- so I've improved it a little bit.

- `Ease.PingPong` is now a static class;
- `Ease.Parabola` is now under `Ease.PingPong`;
- the original `Ease.PingPong` is now `Ease.PingPong.Linear`;
- added a _**Fancy™SineWave**_ -- `Ease.PingPong.Sine`.

### CODE REVIEW

File: Scripts/Easings/PingPong.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+			public static float Parabola(float x)
+			{
+				var sqrRoot = 2f * x - 1f;
+				return 1f - sqrRoot * sqrRoot;
+			}
+
+			public static float Sine(float x)
+			{
+				return (Mathf.Cos(x * PI2 + Mathf.PI) + 1f) * 0.5f;
```
Comment (FancySensei) [inline Scripts/Easings/PingPong.cs:24]: I call it "Sine" but actually using Cosine -- what a troll! :trollface:

Comment (FancySensei) [reply Scripts/Easings/PingPong.cs:24]: either `y = (sin(x * 2 * pi - 0.5 * pi) + 1) * 0.5`
or `y = (cos(x * 2 * pi + pi) + 1) * 0.5`
will do the job.

I just wanted to troll. :trollface:

**Comment by FenrirWillow** [comment]
Didn't we have this functionality already? Ping-pong in tweening libraries usually are just once you reach the max value, go back to the beginning value ([example from the Tween.js library](http://tweenjs.github.io/tween.js/examples/10_yoyo.html))

Or is this a sine/cosine wave where the starting value is the baseline for the wave?

**Comment by kkjamie** [comment]
@FenrirWillow Quite often it's easier to achieve the back and forth effect with an easing function rather than hooking reversing it on complete, and playing again.

**Review by kkjamie — APPROVED** [review]

**Review by FenrirWillow — APPROVED** [review]

**Review by borntocompile — APPROVED** [review]

**Pull request merged**